.NET/Microsoft.Recognizers.Text.Number/Hindi/Extractors/NumberRangeExtractor.cs

```diff
@@ -13,7 +13,7 @@ public NumberRangeExtractor(INumberOptionsConfiguration config)
             : base(
                   NumberExtractor.GetInstance(),
                   OrdinalExtractor.GetInstance(),
-                  new BaseNumberParser(new HindiNumberParserConfiguration(config)),
+                  new BaseIndianNumberParser(new HindiNumberParserConfiguration(config)),
```

Comment:
Question: Could BaseIndianNumberParser be unified with BaseNumberParser and language specific behaviour be contained in it's parser configuration object? That would be preferable, if possible.

Reply:
will investigate this

Reply:
will check this